python/ray/dashboard/client/src/api.ts

```diff
@@ -178,3 +178,14 @@ export const getProfilingResultURL = (profilingId: string) =>
   `${base}/speedscope/index.html#profileURL=${encodeURIComponent(
     `${base}/api/get_profiling_info?profiling_id=${profilingId}`
   )}`;
+
+export const launchKillActor = (
+  actorId: string,
+  actorIpAddress: string,
+  actorPort: number
+) =>
+  get<string>("/api/kill_actor", {
```

Comment:
Does this actually return a string? It looks like the `kill_actor` method in `dashboard.py` might return an empty object?